### ISSUE TITLE
Update of aspida version no longer generates code

### ISSUE THREAD

**Issue by dadajam4** [aspida/openapi2aspida#245]
## Description

A recent update to the aspida package causes the version marked `aspida^1.11.0` to resolve to `1.13.1`, resulting in the following error during code generation

```sh
node:internal/modules/cjs/loader:1096
  const err = new Error(`Cannot find module '${request}'`);
              ^

Error: Cannot find module '/Users/dadajam4/projects/sample/node_modules/.pnpm/openapi2aspida@0.21.0/node_modules/aspida/dist/commands.js'
    at createEsmNotFoundErr (node:internal/modules/cjs/loader:1096:15)
    at finalizeEsmResolution (node:internal/modules/cjs/loader:1089:15)
    at resolveExports (node:internal/modules/cjs/loader:565:14)
    at Module._findPath (node:internal/modules/cjs/loader:634:31)
    at Module._resolveFilename (node:internal/modules/cjs/loader:1061:27)
    at Module._load (node:internal/modules/cjs/loader:920:27)
    at Module.require (node:internal/modules/cjs/loader:1141:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at Object.<anonymous> (/Users/dadajam4/projects/sample/node_modules/.pnpm/openapi2aspida@0.21.0/node_modules/openapi2aspida/dist/getConfig.js:3:18)
    at Module._compile (node:internal/modules/cjs/loader:1254:14) {
  code: 'MODULE_NOT_FOUND',
  path: '/Users/dadajam4/projects/sample/node_modules/.pnpm/openapi2aspida@0.21.0/node_modules/aspida/package.json'
}
```

I am guessing that this is probably due to the fact that the path to `commands.js` has changed due to the aspida package being ESM compliant.
It seems to have already been fixed in the following commit, but when do you plan to release this one?

Thanks for this great library, it always helps me in my development. Thank you very much.

## Environment

- **Package version:**
  
  `v0.21.0`
- **OS:**
  
  - [ ] Linux
  - [ ] Windows
  - [x] macOS
- **Node.js version:**
  
  `v18.13.0`
- **npm version:**
  
  `8.19.3`

## Additional context

It is installed with pnpm. Version is 8.5.0.

**Comment by solufa** [comment]
fixed in v0.22.0

**Comment by dadajam4** [comment]
@solufa 
I was surprised at how quickly you responded. Thank you!
